pkg/cmd/build/buildkit.go

```diff
@@ -289,12 +289,12 @@ func solveBuild(ctx context.Context, c *client.Client, opt *client.SolveOpt, pro
 				oktetoLog.Debugf("could not create console from file: %s ", err)
 			}
 			go func() {
-				if err := progressui.DisplaySolveStatus(context.TODO(), "", c, oktetoLog.GetOutputWriter(), ttyChannel); err != nil {
+				if err := progressui.DisplaySolveStatus(context.TODO(), "", nil, w, plainChannel); err != nil {
```

Comment:
Could we include some comment in the code? It's not intuitive that we have to pass the `plainChannel` to a function within a case for `ttyFormat`

Reply:
+1. Perhaps we can also include why we use tty/plain and now we are swapping